core/src/main/java/com/netflix/conductor/core/execution/SystemTaskType.java

```diff
@@ -21,19 +21,16 @@
 import java.util.HashSet;
 import java.util.Set;
 
-import com.netflix.conductor.core.execution.tasks.Decision;
-import com.netflix.conductor.core.execution.tasks.ExclusiveJoin;
-import com.netflix.conductor.core.execution.tasks.Fork;
-import com.netflix.conductor.core.execution.tasks.Join;
-import com.netflix.conductor.core.execution.tasks.WorkflowSystemTask;
+import com.netflix.conductor.core.execution.tasks.*;
```

Comment:
Would it make more sense to import specific task instead of including all the tasks?